README.md

```diff
@@ -57,13 +57,30 @@ helm dependency update <directory-containing-project-chart>
 Then can run a dry-run upgrade to see the effect:
 
 ```bash
-helm upgrade --dry-run --namespace my-namespace <release-name> <directory-containing-project-chart> --values=<values-file>
+helm upgrade \
```

Comment:
I made these changes to the main `readme` because the instructions are slightly wrong, in that the `--values` argument needs to come before the release name and directory args.

Reply:
this is from the help, see `helm upgrade --help` - although both ways may work.

```
Usage:
  helm upgrade [RELEASE] [CHART] [flags]
```